### PR TITLE
Adds round op and primitive

### PR DESCRIPTION
**Pull request by angeloskath**
Nothing too fancy, follows numpy in the way we handle the `decimals` argument.

Possible caveats:
- It performs round away from 0 unlike numpy which rounds to even (sounds weird tbh)
- Round with provided decimals can lead to infs and nans if decimals is a large number or if the number is very large because it is done by `round(s * x) / s`

### CODE REVIEW

**Comment by angeloskath** [comment]
The build error on linux was fixed by #202 so it should build fine when merged.

**Comment by awni** [comment]
> rounds to even 

What?? That is so weird..

File: python/src/ops.cpp [hunk 1]
```diff
@@ -2922,4 +2922,30 @@ void init_ops(py::module_& m) {
...
+        Bascially performs:
+
+        .. code-block:: python
+
+          s = 10**decimals
+          x = round(x * s) / s
+
+        Args:
+          a (array): Input array
+          decimals (int): Number of decimal places to round to. (default: 0)
```
Comment (awni) [inline python/src/ops.cpp:2949]: For the sake of consistency, could you add the `Returns:` section?

**Review by awni — APPROVED** [review]
Lovely! Minor comment + could you add `round` to the `ops.rst`?

**Pull request merged**